infoextract.js

```diff
@@ -251,7 +251,14 @@ module.exports = {
 					this.getManyPreviews(playlist).then(previews => resolve(previews));
 				}).catch(err => {
 					console.error("Failed to compile add preview: error getting playlist:", err);
```

Comment:
We should change the logging here a little so that we only see stack traces if there is actually a problem